<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>897</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>650</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="maximumSize">
       <size>
        <width>122</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="title">
       <string>GroupBox</string>
      </property>
      <widget class="QListWidget" name="lines">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>20</y>
         <width>121</width>
         <height>231</height>
        </rect>
       </property>
       <property name="sortingEnabled">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>300</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>ax**2 + bx**1 +c</string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="aV">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>320</y>
         <width>62</width>
         <height>22</height>
        </rect>
       </property>
       <property name="minimum">
        <double>-99.989999999999995</double>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="bV">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>350</y>
         <width>62</width>
         <height>22</height>
        </rect>
       </property>
       <property name="minimum">
        <double>-99.989999999999995</double>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="cV">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>380</y>
         <width>62</width>
         <height>22</height>
        </rect>
       </property>
       <property name="minimum">
        <double>-99.989999999999995</double>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>320</y>
         <width>16</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>a:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>350</y>
         <width>16</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>b:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>380</y>
         <width>16</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>c:</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_add">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>260</y>
         <width>41</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>add</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_del">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>260</y>
         <width>41</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>del</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="legned">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>410</y>
         <width>73</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Legend</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
       <property name="tristate">
        <bool>false</bool>
       </property>
      </widget>
     </widget>
    </item>
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="MW_UQE" name="f1" native="true">
        <property name="minimumSize">
         <size>
          <width>502</width>
          <height>400</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QScrollBar" name="hsb_f1">
        <property name="maximum">
         <number>50</number>
        </property>
        <property name="singleStep">
         <number>1</number>
        </property>
        <property name="pageStep">
         <number>1</number>
        </property>
        <property name="value">
         <number>25</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>897</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MW_UQE</class>
   <extends>QWidget</extends>
   <header>MatplotlibWidget</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
